current_pos: [5.544445, 5.544445, 0.0]
current_vel: [+0.3, 0.0, 0.0]
target: [1.1582, 2.0247, 0.0]

behavior: 3
# FLEE 0
# SEEK 1
# STOP 2
# ARRIVAL 3